src/Components/Image/index.js

```diff
@@ -11,6 +11,8 @@ const Image = ({
   caption = false,
   fullHeight: isFullHeight = false,
   maxHeight = "auto",
+  className = "",
+  useLazy = true,
```

Comment:
Porbably need to return to the old method of div bg image and onload. LazyImageLoader was giving some issues